example-mojaloop-backend/Chart.yaml

```diff
@@ -25,34 +26,23 @@ dependencies:
       - backend
       - kafka
     version: 19.0.2
-  ## Central-ledger backend
+  ## mysql database
   - name: mysql
-    alias: centralledger-mysql
-    condition: centralledger-mysql.enabled
+    alias: mysql
+    condition: mysql.enabled
     repository: https://charts.bitnami.com/bitnami
     tags:
       - mojaloop
       - dependency
       - backend
       - mysql
       - centralledger
-    version: 9.4.1
-  ## Account-lookup-service backend
-  - name: mysql
-    alias: account-lookup-mysql
-    condition: account-lookup-mysql.enabled
-    repository: https://charts.bitnami.com/bitnami
-    tags:
-      - mojaloop
-      - dependency
-      - backend
-      - mysql
       - account-lookup
     version: 9.4.1
   ## Bulk backend
   - name: mongodb
-    alias: centralledger-obj
-    condition: centralledger-obj.enabled
+    alias: cl-mongodb
```

Comment:
@mdebarros yes definitely ! However again lets do that as a separate story , so we can get this bunch of changes in because I think that will require a number of changes to container config and container init and that would be another round of reviews ... ?

Reply:
I would prefer to not do this seperately.

We should try be consistent.

Reply:
@mdebarros : ok I will leave this unresolved and look at it after I have done all the other stuff.

Reply:
Created a story for this: 

Separate story for MongoDB consolidation --> https://github.com/mojaloop/project/issues/3104